Scribe/AboutTab/AboutViewController.swift

```diff
@@ -1,42 +1,47 @@
 //
-//  AboutViewController.swift
+//  Copyright (C) 2023 Scribe
```

Comment:
Since the project is based on the GNU license, it is advised by the license itself to add this header on top of each code file.

I'm adding it to new files and files I'm refactoring only, doing a complete pass would add nothing and generate extra commit noise, it can be done over the tilme.

Reply:
I'll make an issue for this, @damien-rivet! Thanks for bringing this to my attention :)

Reply:
See #391 😊